Userland/Libraries/LibManual/SectionNode.cpp

```diff
@@ -22,6 +22,8 @@ ErrorOr<NonnullRefPtr<SectionNode>> SectionNode::try_create_from_number(StringVi
     if (!maybe_section_number.has_value())
         return Error::from_string_literal("Section is not a number");
     auto section_number = maybe_section_number.release_value();
+    if (section_number == 0)
+        return Error::from_string_literal("Section numbers start from 1, you have provided 0.");
```

Comment:
This error message ends in a period, but the others do not. Also, I'd be perfectly fine with a generic "Invalid section number" for `section_number < 1 || section_number > number_of_sections`

Reply:
That is, my main issue is that it's a super specific error message for a single value instead of a generic one, which is what I think most people would expect.

Reply:
Sure, I will sort that out after work